### PR TITLE
[Doc] Fix TestContext JsDocs

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: packages/ra-test/src/TestContext.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const dataProviderDefaultResponse = { data: null };
  * @example
  * // in a react testing-library test, using jest.
  * const utils = render(
- *     <TestContext initialState={{ admin: { resources: { post: { data: { 1: {id: 1, title: 'foo' } } } } } }}>
+ *     <TestContext initialState={{ admin: { resources: { posts: { data: { 1: {id: 1, title: 'foo' } } } } } }}>
```
Comment (djhi) [inline packages/ra-test/src/TestContext.tsx:58]: Why?

Comment (WiXSL) [reply packages/ra-test/src/TestContext.tsx:58]: `posts` are the resources, `post` indicate only one resource

Comment (djhi) [reply packages/ra-test/src/TestContext.tsx:58]: I guess that's more coherent with the other examples but nothing prevent you to name your resources `post`.

**Review by djhi — APPROVED** [review]
Thanks!

**Pull request merged**